shared/src/business/useCases/getCaseInteractor.ts

```diff
@@ -7,23 +7,31 @@ import {
   isAuthUser,
 } from '@shared/business/entities/authUser/AuthUser';
 import { getCaseByDocketNumber } from '@web-api/persistence/postgres/cases/getCaseByDocketNumber';
+import { getWorkItemsByDocketNumber } from '@web-api/persistence/postgres/workitems/getWorkItemsByDocketNumber';
+import { UIDocketEntry } from '@shared/business/entities/DocketEntry';
 
 export const getCaseInteractor = async (
   applicationContext: ServerApplicationContext,
   { docketNumber }: { docketNumber: string },
   authorizedUser: UnknownAuthUser,
-) => {
+): Promise<UICase> => {
```

Comment:
The type here is a bit of a lie since it could be a `RawPublicCase` or `RawRestrictedCase` ... but it's no more of a lie than we already have given that we were saying that the client-side `caseDetail` was always a `RawCase`. (Now we say it is a `UICase`.)

The idea with `UIDocketEntry` and `UICase` is the following: on the UI, we need certain work item info attached to docket entries. I thought about a variety of approaches, including adding these attributes to `DocketEntry` directly or returning work items separately to stitch together the data on the front end. The approach I settled on: attach the work item data we need to an instance of a new type, a `UIDocketEntry`, which represents data that we need on the UI but not the back end.